README.md

```diff
@@ -104,3 +104,28 @@ Within 10 minutes of merging to master, it deploys automatically.
 [12]: https://gohugo.io/getting-started/installing/
 [13]: https://golang.org/doc/install
 [14]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
+
+## Docker development
+
+Prerequsites:
+- Running Monterey OSX
+- Docker Desktop >= 4.7.1 is installed
+- At least 6GB of RAM is dedicated towards Docker for Mac
+  1. Open the Docker for Mac app dashboard
+  2. Click the gear icon
+  3. Click Resources
+  4. The memory slider should be set to 6GB
+- VirtioFS is enabled
+  1. Open the Docker for Mac app dashboard
+  2. Click the gear icon
+  3. Click Experimental Features
+  4. Click Enable VirtioFS accelerated directory sharing
```

Comment:
```suggestion
  4. Click Enable VirtioFS accelerated directory sharing
  5. Click Apply & Restart
```

Reply:
I tried `make start-docker FULL_BUILD=true` and got lots of errors like this:
```
Building the documentation with ALL external content:
docs_1  | No GITHUB TOKEN was found. Skipping any data sync that relies on pulling from web.
docs_1  | Add all source repositories in the same parent folder as the documentation/ folder to build the full doc locally.
docs_1  | Or use the 'make start-no-pre-build' command to build the doc without any external content.
docs_1  | WARNING: No local version of datadog-plugin found, no GITHUB_TOKEN available. Documentation is now in degraded mode
docs_1  | WARNING: No local version of aws-sdk-go found, no GITHUB_TOKEN available. Documentation is now in degraded mode
docs_1  | WARNING: No local version of dd-go found, no GITHUB_TOKEN available. Documentation is now in degraded mode
```

Reply:
It should from the steps in the readme https://github.com/DataDog/documentation/blob/ccole/docker-local-build/README.md#how-to-run-documentation-inside-a-docker-container

I believe its a combination of making sure your `makefile.config` has the entries that exist in `makefile.config.example` especially `GITHUB_TOKEN` and `FULL_BUILD`

Generally right now the idea is that we can merge this pr and everyones local dev works as is but we can try evaluate using the docker image alongside this and see what we think.

Reply:
Okay, I figured out what I'd done wrong (messed up my GITHUB_TOKEN in the config file). The full build seems to be working now, but it takes a really long time. If this is meant to be a testing phase alongside the normal local dev then I'm going to go ahead and approve 👍